test/user.test.js

```diff
@@ -2442,7 +2442,7 @@ describe('User', function() {
     it('handles subclassed user with no accessToken relation', () => {
       // setup a new LoopBack app, we don't want to use shared models
       app = loopback({localRegistry: true, loadBuiltinModels: true});
-      app.set('_verifyAuthModelRelations', false);
+      app.set('_warnOnBadAuthModelsSetup', false);
```

Comment:
it would mean that it's the exact same name as the app method 
`app. _verifyAuthModelRelations()`
is it problematic?